docs/index.rst

```diff
@@ -12,26 +12,55 @@
 
 ----------------------------------
 
-.. include:: quickstart.rst
-    :start-line: 1
+.. toctree::
+   :maxdepth: 1
+   :caption: Concepts
 
+   concepts/tutor 
+   concepts/openedx
+   
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
    :caption: User guide
 
-   gettingstarted
+   install
+   deployment
    run
    configuration
    plugins/index
-   reference/index
-   tutorials/index
```

Comment:
I understand they have a different format, but there is a lot of important documentation that *only* exists in a tutorial.

If we take "Creating a Tutor Plugin" as an example, [in the current docs](https://docs.tutor.overhang.io/tutorials/plugin.html) it is nested under
* User Guide
  * Tutorials
    * Open edX customization
      * Creating a Tutor plugin

In the reorg [I moved it to](https://brian-smith-tcril.github.io/tutor/developing/plugins/creating.html)
* Developer Guide
  * Tutor plugin development
    * Creating a Tutor plugin
    
My thought process here was if I were coming to the docs with the goal of creating a tutor plugin, I would have no idea that I should look that deep into the Tutorials section. In fact, that exact scenario is what inspired me to start working on this reorg in the first place :joy: 

I understand the desire to separate them based on the format of the information, but it seems that is currently happening at the expense of discoverability.

I'm very open to ideas though! Any thoughts on how to keep the format distinction without hiding the information away?

Reply:
> If we move to having top level "Tutorials" I think we could make that beeline with
> 
>     Tutorials
>         Plugins
>             Creating a Tutor plugin

I agree, let's do that.

I also agree that in many cases the information is indeed hard to find -- the nightly docs is a good example that you pointed out. But this is as much a problem of reorganising the section as putting more words in the docs. Can I suggest that you list the place in the docs that, in your opinion, lack some information? You can either create a GitHub issue or add a big TODO in the file where you would expect to find more words. Depending on the amount of work, we might resolve these issues in this PR or later.

Reply:
The more I think about this the less I like it. By splitting up information at the top-level based on how it's presented, navigation becomes much more convoluted.

I understand the principles of Diátaxis suggest organizing docs into Tutorials/How-Tos/Explanation/Reference, but I think that separation makes sense *within* other categories.

If we take the 4 types and turn them into user statements, we arrive at

| Type | Statement |
| - | - |
| Tutorials | I want to learn about _____. |
| How-To Guides | How do I do something related to _____? |
| Explanation | I want to better understand _____. |
| Reference | I need information about _____. |

If we fill in the blank with "Tutor," then it makes sense to have a top level split based on Diátaxis type. I don't, however, think most people coming to the docs would fill in the blank that way.

I know I came to the docs filling in that blank with "Tutor plugins." I know plenty of people come to the docs filling in that blank with "running Open edX."

With this in mind, I think it makes sense to make sure the high level categories line up with how people would "fill in the blank," and then find a way to clearly differentiate the information by Diátaxis type within said categories.

Thinking about this in practical terms, I've asked myself, "what should be visible in the navigation sidebar when viewing this?" I believe people reading a tutorial about plugins are more likely to want to see more information about plugins in the sidebar than other tutorials.

I'm very open to other thoughts and feelings on this, I just wanted to make sure I expressed mine clearly before making changes I was unhappy with.

Reply:
Hey, sorry about the lack of reply. I was busy with the Palm upgrade for the past couple of days.

I think you really shouldn't be making changes that you're unhappy with. You want to make it as easy as possible for users to discover the plugins tutorial, and that's perfectly reasonable. You argue that many users navigate their way through the docs mostly by using the navigation sidebar, and that the first point of entry for users interested in plugins is the "Plugins" section -- again, perfectly reasonable.

What I want to avoid is to distribute the tutorials across different sections. There are a two main reasons for that:

1. Some tutorials do not belong to one specific section.
2. The Tutorials are in some way a kitchen sink where we can dump information about many different topics (similar to the [Django "How-To" guides](https://docs.djangoproject.com/en/4.2/howto/)).

To reconcile your position and mine, can we add an entry to the plugins TOC that would send users directly to the plugins tutorial?

Reply:
Hello there :D I was checking the doc for an issue I have assigned, and finding the info about creating a tutor plugin was difficult. I think the @regisb idea of adding an entry in Plugin that redirects to plugin tutorials is a good idea.

Btw @brian-smith-tcril, thanks for this PR is fantastic.

Reply:
> To reconcile your position and mine, can we add an entry to the plugins TOC that would send users directly to the plugins tutorial?

What would the navigation sidebar look like in this scenario? Would we have both plugins and tutorials expanded when someone is reading the plugins tutorial?